### PR TITLE
Intro to K3S

### PR DESCRIPTION
**Pull request by chanaSlumianski**
A short description of the K3S system in markdown form.

### CODE REVIEW

**Comment by chanaSlumianski** [comment]
I would like to know if my task was completed as required.

**Comment by bdsoha** [comment]
Your task is complete when it is merged.

Please move your files into a subdirectory.

**Comment by chanaSlumianski** [comment]
ok

**Comment by chanaSlumianski** [comment]
do you see my commits?

**Comment by bdsoha** [comment]
Please move your files into a subdirectory called `k3s`.

Change the readme to `README.md`.

**Comment by chanaSlumianski** [comment]
> Please move your files into a subdirectory called `k3s`.
> 
> Change the readme to `README.md`.

i did clone to my branch and I don't see any files...

**Comment by bdsoha** [comment]
Move into your cloned directory with `cd`

**Comment by chanaSlumianski** [comment]
done

File: registration
Comment (bdsoha) [inline registration:1]: Why is there a submodule here?

**Comment by chanaSlumianski** [comment]
what do you mean **submodule** ?

**Comment by chanaSlumianski** [comment]
ah I got it...
i'm deleting

**Comment by chanaSlumianski** [comment]
done

File: k3s/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Use an existing docker image as a base
+# FROM pandoc/core:latest
+FROM pandoc/latex:2.14.0.1
+
+WORKDIR /k3s
```
Comment (bdsoha) [inline k3s/Dockerfile:5]: Is there a need to define `WORKDIR`.

File: k3s/README.MD [hunk 2]
```diff
@@ -0,0 +1,33 @@
+## _Markdown document describing k3s_
```
Comment (bdsoha) [inline k3s/README.MD:1]:
```suggestion
# K3S
```

File: k3s/README.MD [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+- It eliminates the need to expose a port on Kubernetes worker nodes for the kubelet API by exposing this API to the Kubernetes control plane nodes over a websocket tunnel.
+
+## Why should I use it?
+
+- Production ready and secure.
+- easy to install.
+- half the memory. 
+- all in a binary less than 100 MB.
+- certified Kubernetes distribution designed for production workloads  in unattended,
+remote locations or inside IoT ("Internet of Things" - Devices that are connected and can communicate with other devices) appliances.
```
Comment (bdsoha) [inline k3s/README.MD:21]:
```suggestion
- Easy to install.
- Half the memory. 
- All in a binary less than 100 MB.
- Certified Kubernetes distribution designed for production workloads  in unattended,
remote locations or inside IoT *("Internet of Things" - Devices that are connected and can communicate with other devices)* appliances.
```

File: k3s/README.MD [hunk 4]
```diff
@@ -0,0 +1,34 @@
+## _K3S_
```
Comment (bdsoha) [inline k3s/README.MD:1]:
```suggestion
# K3S
```

**Comment by chanaSlumianski** [comment]
done

File: k3s/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,6 @@
+
+FROM pandoc/latex:2.14.0.1
```
Comment (bdsoha) [inline k3s/Dockerfile:2]:
```suggestion
FROM pandoc/latex:2.14.0.1
```

**Comment by bdsoha** [comment]
**🎉 Congrats on your first merged PR.**

You will soon be added as a collaborator to this repo.
In the meantime, please prepare a short presentation *(in English)* describing:

- The task in general.
- Things you learned along the way.
- Tips for other team-members for future tasks.

**Comment by bdsoha** [comment]
@chanaSlumianski Please do the following:
- Remove https://github.com/chanaSlumianski/feature-my-task-name.
- Change your profile name to `Chana Slumianski` using title-case.

**Comment by bdsoha** [comment]
You have been sent an email invitation to the [SkyVar Organization](https://github.com/skyvar).
Please open your cooperate email and finish the registration process.

**Comment by chanaSlumianski** [comment]
סיימתי ...
מה הלאה?
________________________________
מאת: Dov Benyomin Sohacheski ***@***.***>
‏‏נשלח: יום שני 20 מרץ 2023 09:06
‏‏אל: skyvar/registration ***@***.***>
עותק: C. Slumianski ***@***.***>; Mention ***@***.***>
‏‏נושא: Re: [skyvar/registration] Intro to K3S (PR #29)


You have been sent an email invitation to the SkyVar Organization<https://github.com/skyvar>.
Please open your cooperate email and finish the registration process.

**Pull request merged**